osidb/tests/factories.py

```diff
@@ -492,19 +504,23 @@ class Params:
             lambda f: f.cve_id if f.source == Snippet.Source.NVD else "GHSA-0001"
         )
         url = factory.LazyAttribute(
-            lambda f: f"https://nvd.nist.gov/vuln/detail/{f.ext_id}"
-            if f.source == Snippet.Source.NVD
-            else f"https://osv.dev/vulnerability/{f.ext_id}"
+            lambda f: (
+                f"https://nvd.nist.gov/vuln/detail/{f.ext_id}"
+                if f.source == Snippet.Source.NVD
+                else f"https://osv.dev/vulnerability/{f.ext_id}"
+            )
```

Comment:
I am thinking what is different here, maybe a different version of **black**?

Reply:
Yes, for some reason the `make format` command was throwing an `invalid ssl certificate` error when trying to install black dependencies so I just ran my local black instance (with the same arguments), I've fixed the error of `make format` and is working now

Reply:
I do not have any serious objections against it until the **black** here in the pipeline is satisfied.